src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) firstLeg * secondLeg / 2;
+    }
+
+    @Override
```

Comment:
There's an empty line at the beginning of the `draw` method implementation. Please remove it to follow the coding standards.